### DISCUSSION TITLE
course assessment is failing while sending the content for review

### DISCUSSION THREAD

**Discussion by srikanth-6520** [Sunbird-Ed/Community#598] [Bugs] [answered]
@princegupta1131 @amith , Course assessment is throwing an error, while sending the content for review, we have shared screenshot along here.

![image](https://github.com/Sunbird-Ed/Community/assets/133200106/6f8e3e04-5f2a-471b-8d1e-6de885cf93ed)

**Comment by amitpriyadarshi** [comment]
@srikanth-6520 as asked, please share object read api response and also body of the ecml content. It seems some of the attached asset is having invalidf file url. 
Also, please try to create some other ECML content.
cc: @Krishnaj20 @vinukumar-vs

**Comment by amitpriyadarshi** [comment]
@srikanth-6520 ECML content is getting created properly. It seems you are using any asset which has invalid file url.  Please create a fresh Resource and validate. It should work properly.

**Reply by srikanth-6520** [reply]
@amitpriyadarshi , if you don't mind can you share the curl command for ecml content .

**Comment by amitpriyadarshi** [comment, marked as answer]
@srikanth-6520 as per our discussion, the issue is because of wrong configuration for baseUrl in content service, because of which the url available in body are not formed properly and while validating it is throwing invalid fileUrl. Please connect with @tsprasath or @gandham-santhosh and correct the configuration.

cc: @Krishnaj20 @vinukumar-vs

**Reply by srikanth-6520** [reply]
@amitpriyadarshi , i have found the issue , the actual issue with one of the file was missed from content-plugins , i have fixed it , thank you , i can able to publish the content now , so we can close it.

**Comment by amitpriyadarshi** [comment]
@srikanth-6520 please add the content service log (which were having stack trace) while calling review api. This will help others to analyse similar issues.
